$Thinkum$

Update ilisp-snd.el to use newer Emacs Lisp backquote syntax

--- ilisp-snd.el.orig	2002-12-03 20:51:13.000000000 +0000
+++ ilisp-snd.el	2019-10-26 08:51:00.683820317 +0000
@@ -393,16 +393,16 @@
 	    (comint-send 
 	     (ilisp-process) binary
 	     t nil 'binary nil 
-	     (` (lambda (error wait message output last)
+	     `(lambda (error wait message output last)
 		  (if (or error
 			  (not (string-match "\"[^\"]*\"" output)))
 		      (progn
 			(lisp-display-output output)
 			(abort-commands-lisp "No binary"))
-		    (setq (, var)
+		    (setq ,var
 			  (substring output
 				     (1+ (match-beginning 0))
-				     (1- (match-end 0))))))))))))
+				     (1- (match-end 0)))))))))))
 
 ;;;
 (defun ilisp-done-init ()
